examples/with-yoga/package.json

```
@@ -0,0 +1,20 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@gympass/yoga": "latest",
```

```suggestion
    "@gympass/yoga": "^7.28.0",
```
Should we lock this to a semver major in case breaking changes are made in the upstream package?